api/src/controllers/young/inscription2023.js

```diff
@@ -445,6 +445,51 @@ router.put("/goToInscriptionAgain", passport.authenticate("young", { session: fa
   }
 });
 
+router.put("/profil", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const { error, value } = Joi.object({
+      firstName: validateFirstName().trim().required(),
+      lastName: Joi.string().uppercase().trim().required(),
+      email: Joi.string().lowercase().trim().email().required(),
+    }).validate(req.body, { stripUnknown: true });
```

Comment:
Franchement ça peut passer : 


```suggestion
    const profilSchema = {
      firstName: validateFirstName().trim().required(),
      lastName: Joi.string().uppercase().trim().required(),
      email: Joi.string().lowercase().trim().email().required(),
    };
    const keyList = Object.keys(profilSchema);
    const { error, value } = Joi.object(profilSchema).validate(req.body, { stripUnknown: true });
```

Reply:
oui  tu m'as convaincu